.github/workflows/ci.yml

```diff
@@ -100,6 +101,7 @@ jobs:
     runs-on: windows-latest
     env:
       LIBCLANG_PATH: 'C:\Program Files\LLVM\bin'
+      RUSTFLAGS: ""
```

Comment:
Why?

Reply:
Was about to comment about that :D With nothing there, there was an error that `-A deref-nullptr` was not recognised by `rustc`, with only `--deny=warnings` the errors were still showing. Not sure why

Reply:
I guess maybe l102 overwrites L11.

Reply:
Yes, I thought that as well, that putting an empty string here would empty the `RUSTFLAGS` defined before. It still failed but that seems to be for another reason?